recipe/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+if [[ ("${target_platform}" == "win-64" && "${build_platform}" == "linux-64") ]]; then
+  # we need to add the generate-import-lib feature since otherwise
+  # maturin will expect libpython DSOs at PYO3_CROSS_LIB_DIR
+  # which we don't have since we are not able to add python as a host dependency
+  cargo add pyo3 \
```

Comment:
This only works if "pyo3" is not an existing dependency, which is not the case with this "Cargo.toml".